lib/Context.js

```diff
@@ -27,9 +28,18 @@ class Context {
     })
   }
 
-  out (predicate) {
-    return this.matchProperty(toArray(this.term), predicate, null, toArray(this.graph), 'object').map(subject => {
-      return this.clone({ value: subject })
+  out (predicate, options) {
```

Comment:
Why not using a destructuring argument? Would make the rest of the code more readable.